aict_tools/plotting.py

```diff
@@ -236,3 +246,21 @@ def plot_feature_importances(model, feature_names, ax=None, max_features=20):
     if len(feature_names) > max_features:
         ax.set_title('The {} most important features'.format(max_features))
     ax.figure.tight_layout()
+
+def plot_true_delta_delta(data_df, ax=None,  source_az='source_azimuth', source_zd='source_zenith', 
+    pointing_az='pointing_azimuth', pointing_zd='pointing_zenith', focal_len='focal_length', x_str='x', y_str='y', psi_str='psi'):
+    
+    df = data_df.copy()
+    df['source_x'], df['source_y'] = horizontal_to_camera_cta_simtel(
```

Comment:
This should also work for FACT, not only for CTA

Reply:
See the code in `train_disp`

Reply:
```
if model_config.coordinate_transformation == 'CTA':
        from ..cta_helpers import horizontal_to_camera_cta_simtel
        source_x, source_y = horizontal_to_camera_cta_simtel(
            az=df[model_config.source_az_column],
            zd=df[model_config.source_zd_column],
            az_pointing=df[model_config.pointing_az_column],
            zd_pointing=df[model_config.pointing_zd_column],
            focal_length=df[model_config.focal_length_column],
        )
```
I believe you mean this code. I will use this but I don't understand why 
```
focal_length=df[model_config.focal_length_column],
```
even works in `train_disp` in the first place. There is no `focal_length_column` in 'config_source_cta.yaml` or any of the other example config files.

Reply:
There are defaults for most config options, see aict_tools.config.py

Reply:
I have just found that `focal_length_column` gets defined for the `class DispConfig` in the configuration script, but I still don't see how it gets the right value assigned to it.

Reply:
Ah, I see. Thx